alternator/executor.cc

```diff
@@ -687,14 +687,14 @@ future<> executor::fill_table_size(rjson::value &table_description, schema_ptr s
     }
     else {
         total_size = co_await _ss.estimate_total_sstable_volume(schema->id(), service::storage_service::ignore_errors::yes);
-
+        const auto timeout = std::chrono::seconds{ _proxy.data_dictionary().get_config().alternator_describe_table_info_timeout_in_seconds() };
```

Comment:
I would squash this patch with the earlier one that introduced the fixed 6 hur timeout.